index.js

```diff
@@ -1 +1,113 @@
 // implement your API here
+const express = require('express');
+const cors = require('cors');
+const db = require('./data/db');
+const server = express();
+server.use(express.json());
+server.use(cors());
```

Comment:
only necessary if you'll start connecting with a react app or other app, but it never hurts to have it brought in just in case instead of forgetting to add it later